disco_aws_automation/disco_metanetwork.py

```diff
@@ -412,7 +412,8 @@ def update_gateways_and_routes(self, desired_route_tuples, dry_run=False):
         current_route_tuples = set()
         if self.centralized_route_table:
             for route in self.centralized_route_table.routes:
-                if route.gateway_id and route.gateway_id != 'local':
+                if route.destination_cidr_block and \
```

Comment:
Do we want to silently skipping when cidr block is missing? Previously it would error out on current_route_tuples?